### PR TITLE
Fix `padWorkflowName()`

### PR DESCRIPTION
**Pull request by vyzaldysanchez**

### CODE REVIEW

File: pkg/capabilities/consensus/ocr3/types/aggregator.go [hunk 1]
```diff
@@ -24,10 +25,16 @@ type Metadata struct {
...
 func (m *Metadata) padWorkflowName() {
-	if len(m.WorkflowName) < 10 {
+	b, err := hex.DecodeString(m.WorkflowName)
+	if err == nil && len(b) < 10 {
+		// Each byte is 2 characters, so we need to pad with 0s
+		neededBytes := append(b, make([]byte, 10-len(b))...)
+		m.WorkflowName = hex.EncodeToString(neededBytes)
+	} else if len(m.WorkflowName) < 10 {
+		// Pad with spaces
 		suffix := strings.Repeat(" ", 10-len(m.WorkflowName))
```
Comment (bolekk) [inline pkg/capabilities/consensus/ocr3/types/aggregator.go:38]: This should never be the case. You also don't really have to hex-decode. Just pad with "0" chars up to length 20.

**Review by bolekk — CHANGES_REQUESTED** [review]

File: pkg/capabilities/consensus/ocr3/types/aggregator.go (hunk 1 above)
Comment (bolekk) [reply pkg/capabilities/consensus/ocr3/types/aggregator.go:38]: cc @krehermann  - cap_encoder.go in core validates that and fails it it's not a valid hex string that decodes to 10 bytes. We should require it here too.

Comment (krehermann) [reply pkg/capabilities/consensus/ocr3/types/aggregator.go]: ok. if it's got to be that then let's move the validation to this struct. can be a follow up change.

code comments aren't sufficient if there are restrictions on the values. and doesn't make sense to have validation far away in a different repo. @vyzaldysanchez can you do that as a follow up (write a Validate method for this struct)

**Review by krehermann — APPROVED** [review]

File: pkg/capabilities/consensus/ocr3/types/aggregator_test.go [hunk 2]
```diff
@@ -16,25 +16,32 @@ func TestMetadata_padWorkflowName(t *testing.T) {
...
 			},
-			want: "          ",
+			want: "00000000000000000000",
+		},
+		{
+			name: "padWorkflowName non-hex string",
+			fields: fields{
+				WorkflowName: "not-hex",
+			},
+			want: "not-hex0000000000000",
```
Comment (bolekk) [inline pkg/capabilities/consensus/ocr3/types/aggregator_test.go:44]: lol, technically correct  ;)

**Review by bolekk — APPROVED** [review]

File: pkg/capabilities/consensus/ocr3/types/aggregator.go (hunk 1 above)
Comment (vyzaldysanchez) [reply pkg/capabilities/consensus/ocr3/types/aggregator.go]: sure thing!

**Pull request merged**